custom_registration/models.py

```diff
@@ -69,7 +69,7 @@ def confirm_user_email(user):
 
 
 def generate_team_invitation_by_email_domain(user):
-    domain = re.search("@([\w.]+)$", user.email).groups()[0]
+    domain = user.email.rsplit('@', 1)[1]
```

Comment:
I guess that's much simpler :) - I didn't realize, but apparently this would be a valid email address according to RFC: `"(blah)user@flo.op"@gmail.com`. I doubt any provider would let you register that though.